guides/common/modules/ref_best-practices-for-products-and-repositories.adoc

```diff
@@ -0,0 +1,29 @@
+[id="best-practices-for-products-and-repositories_{context}"]
+= Best practices for products and repositories
+
+* Use one content type per product and content view, for example {client-content-type} content only.
+ifdef::orcharhino[]
+* Use file repositories for installation media.
+File repositories require a `PULP_MANIFEST` file that you can create using `pulp-manifest /path/to/files`.
+Use local repositories with `\file:///path/to/files` as *Upstream URL* in {Project}, for example for installation media.
+Alternatively, place the installation media file repositories on a web server that can be reached from hosts during provisioning.
++
+include::snip_tip-file-repositories-by-atix.adoc[]
++
+For more information, see xref:Managing_Custom_File_Type_Content_{context}[].
+endif::[]
+* Make file repositories available over HTTP.
+If you set *Protected* to true, you can only download content using a global debugging certificate.
+* Automate the creation of multiple products and repositories by using a Hammer script or an {ansible-docs-url}[Ansible playbook].
+ifdef::katello,satellite,red_hat_enterprise_linux[]
+* For Red Hat content, import your Red Hat manifest into {Project}.
+For more information, see xref:Managing_Red_Hat_Subscriptions_{context}[].
+endif::[]
+ifdef::katello,suse_linux_enterprise_server[]
+* For SUSE content, use the SCC Manager plugin.
+For more information, see xref:Managing_SUSE_Content_{context}[].
+endif::[]
+* Avoid uploading content to repositories with an *Upstream URL*.
+Instead, create a repository to synchronize content and upload content to without setting an *Upstream URL*.
++
+If you upload content to a repository that already synchronizes another repository, there might be unwanted side effects depending on the mirroring policy and content type.
```

Comment:
I am guessing that this is what might happen.

```suggestion
If you upload content to a repository that already synchronizes another repository, the content might be over-written, depending on the mirroring policy and content type.
```

Reply:
Applied with "overwritten".